artillery/add_password.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+input="user_list"
+while IFS= read -r var
+do
+  echo "$var"$(pwgen 20) >> user_list_2
```

Comment:
Je ne suis pas certains de savoir à quoi corresponds ce script :/ et pwgen n'est pas une lib par défaut, il faudrait le préciser dans le readme ou faire un check and install au début du script